modules/www/main.tf

```diff
@@ -472,3 +474,16 @@ resource "aws_ecs_service" "www_ecs_service" {
     aws_lb_listener.www_lb_listener
   ]
 }
+
+module "logging_configs" {
+  source            = "../logging_configs"
+  app_name          = local.app_name
+  environment_name  = var.environment_name
+  task_name         = local.app_name
+  datadog_image     = var.datadog_image
+  datadog_image_tag = var.datadog_image_tag
```

Comment:
I think we'll probably need to swap this one out for the docker image tag of the nerves hub app.

Reply:
Done ✅